submissions/ifeanyiLawson/html-forms/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Landing Page</title>
+    </head>
+    <body>
+    <nav>
+        <ul>
+            <li><a href="index.html">Product Page</a></li>
+            <li><a href="../html-forms/contactformpage.html">Contact Form</a></li>
+        </ul>
+    </nav>
+    <h1>Product Page</h1>
+        <img src="images/IMG_1554.jpg" width="500" height="500">
+        <p><a href="#index.html"><strong>AIRFORCE NIKE</strong></a></p>
```

Comment:
•	A relative URL - points to a file within a website (like href="default.htm")
•	Link to an element with a specified id within the page (like href="#section2")
A hash - `#` within a hyperlink specifies an HTML element id to which the window should be scrolled.
href="#some-id" would scroll to an element on the current page such as <div id="some-id">.